pkg/infrastructure/services/resources.go

```diff
@@ -197,11 +200,10 @@ func (s *serviceDeployer) applyKafkaConfigurations(deployment *appsv1.Deployment
 	if len(URI) > 0 {
 		framework.SetEnvVar(enableEventsEnvKey, "true", &deployment.Spec.Template.Spec.Containers[0])
 		for _, kafkaTopic := range s.definition.KafkaTopics {
-			framework.SetEnvVar(fromKafkaTopicToQuarkusEnvVar(kafkaTopic), URI, &deployment.Spec.Template.Spec.Containers[0])
-		}
-		for _, kafkaEnv := range quarkusBootstrapEnvVars {
-			framework.SetEnvVar(kafkaEnv, URI, &deployment.Spec.Template.Spec.Containers[0])
+			appProps[fromKafkaTopicToQuarkusAppProp(kafkaTopic)] = URI
 		}
+		framework.SetEnvVar(quarkusBootstrapEnvVar, URI, &deployment.Spec.Template.Spec.Containers[0])
+		appProps[QuarkusBootstrapAppProp] = URI
```

Comment:
Shouldn't be applied SpringBoot property here in case of SpringBoot runtime?

Reply:
Corrected. Thanks.